### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by jjimenez0611**
## Scope

The scope for this PR is create new models

## Implementation

Create new models

### CODE REVIEW

File: app/src/main/java/com/effeta/miparroquiaandroid/views/models/Eucharist.kt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.effeta.miparroquiaandroid.views.models
+
+import android.location.Geocoder
+import android.location.Location
+
+/**
+ * Created by jjimenez on 2/2/18.
+ */
+class Eucharist (val mImage : String, val mName : String, val mUbication: Location) {
```
Comment (CodingAleCR) [inline app/src/main/java/com/effeta/miparroquiaandroid/views/models/Eucharist.kt]: Rename to Church

File: app/src/main/java/com/effeta/miparroquiaandroid/views/models/Mass.kt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package com.effeta.miparroquiaandroid.views.models
+
+import java.sql.Timestamp
+
+
+/**
+ * Created by jjimenez on 2/2/18.
+ */
+class Mass(val mEucharist: String, val mHour: Timestamp, val mPriestName: String) {
```
Comment (CodingAleCR) [inline app/src/main/java/com/effeta/miparroquiaandroid/views/models/Mass.kt]: Rename to Eucharist

File: app/src/main/java/com/effeta/miparroquiaandroid/views/models/Eucharist.kt (hunk 1 above)
Comment (jjimenez0611) [reply app/src/main/java/com/effeta/miparroquiaandroid/views/models/Eucharist.kt]: ready

File: app/src/main/java/com/effeta/miparroquiaandroid/views/models/Mass.kt (hunk 2 above)
Comment (jjimenez0611) [reply app/src/main/java/com/effeta/miparroquiaandroid/views/models/Mass.kt]: Ready

**Pull request merged**